### PR TITLE
ITA: add and translate two missing sections

### PR DESCRIPTION
**Pull request by LorenzoAncora**
This PR adds two sections which were missing from the Italian translation:

- Is “v1.2.3” a semantic version?
- Is there a suggested regular expression (RegEx) to check a SemVer string?

Both are full Italian translations.

## PR Checklist

- [X] This PR doesn't modify the files in the `spec` dir.
- [X] This PR passes `npm run lint`.
- [X] (Optional) If you're willing to help reviews about your language's translations, add your name to the `TRANSLATORS.md`.

### CODE REVIEW

File: TRANSLATORS.md
Comment (LorenzoAncora) [inline TRANSLATORS.md:1]: af02325ecdd1aec954cb0feb1462caf3da753b35 fixes merge conflict with Serbian.

**Comment by LorenzoAncora** [comment]
@JohnTitor I'm awaiting for your approval. Feel free to commit directly to my branch if quick fixes are needed.

**Review by JohnTitor — APPROVED** [review]
Sorry for the delay, thank you!

File: lang/it/index.md [hunk 1]
```diff
@@ -271,6 +271,38 @@ No, ma usate il buon senso. Per esempio, una stringa di versione di 255 caratter
 è eccessiva. Inoltre, sistemi specifici possono imporre i loro limiti sulla 
 dimensione.
 
+
+### "v1.2.3" è una versione semantica?
```
Comment (JohnTitor) [inline lang/it/index.md]:
```suggestion
### "v1.2.3" è una versione semantica?
```

**Pull request merged**